packages/components/src/stories/install-guide/ProjectPicker.stories.js

```diff
@@ -215,6 +215,26 @@ RubyVSCode.args = {
   editor: 'vscode',
 };
 
+export const JavascriptVSCode = Template.bind({});
```

Comment:
There's already a `JavaScriptVSCode` story for the Project Picker, so now there are two JS stories. Can you consolidate them into one?